lib/taskbook.js

```diff
@@ -499,6 +499,18 @@ class Taskbook {
     this._save(_data);
     render.successPriority(id, level);
   }
+
+  clean() {
+    const keysToDelete = [];
+
+    Object.entries(this._data).forEach(([id, item]) => {
+      if (item.isComplete) {
+        keysToDelete.push(id);
+      }
+    });
```

Comment:
`Object.entries` can be replaced by `Object.key` so the whole method is compatible with node.js v6:

```js
Object.keys(_data).forEach(id => {
  if (_data[id].isComplete) {
    ids.push(id);
  }
});
```

Reply:
Yes that's true